src/Resources/views/dashboard.html.twig

```diff
@@ -58,11 +58,47 @@
                         <td><a class="btn btn-danger" href="{{ path('karo-io.reject_failed_message', {id: failedMessage.id}) }}">Reject</a></td>
                     </tr>
                 {% else %}
-                    <tr><td colspan="6">Your failure transport is empty. You're killing it!</td></tr>
+                    <tr>
+                        <td colspan="6">Your failure transport is empty. You're killing it!</td>
+                    </tr>
                 {% endfor %}
             </table>
         {% endif %}
     {% endif %}
+
+    {% if statistics.metrics|length %}
+        <h2>Statistics for last 24 hours</h2>
+
+        <h3 class="mt-3">Overall statistics</h3>
+        <div class="card" style="width: 18rem;">
+            <ul class="list-group list-group-flush">
+                <li class="list-group-item">Messages count: {{ statistics.messagesCount }}</li>
+                <li class="list-group-item">Messages per hour: {{ statistics.messagesHandledPerHour }}</li>
+                <li class="list-group-item">Average waiting time: {{ statistics.averageWaitingTime }}</li>
+                <li class="list-group-item">Average handling time: {{ statistics.averageHandlingTime }}</li>
```

Comment:
We're going to need to do some nice time display on all of these. Here's one example:

<img width="320" alt="Screen Shot 2020-03-06 at 12 23 03 PM" src="https://user-images.githubusercontent.com/121003/76106632-3e702180-5fa5-11ea-84c6-d6a4376306e5.png">

Ideally that would be:

```
Average waiting time: 22 minutes
Average handling time: 2.25 seconds
```

Minutes might be rare - but we should handle them I think.

Reply:
it's been such a loooong time i haven't created a TwigExtension ;)